@here/harp-mapview/lib/AnimatedExtrusionHandler.ts

```diff
@@ -0,0 +1,353 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { TileKey } from "@here/harp-geoutils";
+import { ExtrusionFeature } from "@here/harp-materials";
+import { MathUtils } from "@here/harp-utils";
+import { MapView, MapViewEventNames, RenderEvent } from "./MapView";
+import { Tile } from "./Tile";
+
+import * as THREE from "three";
+
+/**
+ * Animation states for extrusion effect
+ */
+export enum AnimatedExtrusionState {
+    None,
+    Started,
+    Playing,
+    Finished
+}
+
+/**
+ * Handles animated extrusion effect of the buildings in [[MapView]].
+ */
+export class AnimatedExtrusionHandler {
+    /**
+     * Animate the extrusion of the buildings if set to `true`.
+     */
+    enabled: boolean = true;
+    /**
+     * Duration of the building's extrusion in milliseconds
+     */
+    duration: number = 750;
```

Comment:
Please initialize the value through a new constant outside of the class: `const DEFAULT_ANIMATION_DURATION = 750;`.